internal/pkg/manifest/env.go

```diff
@@ -85,10 +85,11 @@ func (e *Environment) MarshalBinary() ([]byte, error) {
 
 // EnvironmentConfig defines the configuration settings for an environment manifest
 type EnvironmentConfig struct {
-	Network       environmentNetworkConfig `yaml:"network,omitempty,flow"`
-	Observability environmentObservability `yaml:"observability,omitempty,flow"`
-	HTTPConfig    EnvironmentHTTPConfig    `yaml:"http,omitempty,flow"`
-	CDNConfig     environmentCDNConfig     `yaml:"cdn,omitempty,flow"`
+	Network             environmentNetworkConfig `yaml:"network,omitempty,flow"`
+	Observability       environmentObservability `yaml:"observability,omitempty,flow"`
+	HTTPConfig          EnvironmentHTTPConfig    `yaml:"http,omitempty,flow"`
+	CDNConfig           environmentCDNConfig     `yaml:"cdn,omitempty,flow"`
+	SecurityGroupConfig securityGroupConfig      `yaml:"security_group,omitempty"`
```

Comment:
Sorry Parag this is my bad! would you mind moving back this field to under what you had originally `network.vpc.`?

The main reasoning behind it is that:
1. Consistency between the existing [`network.vpc.security_groups`](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#network-vpc-security-groups) field as this sg affects the connectivity of all services in the VPC.
2. We will be adding also `http.public.security_group` so there is a nice symmetry by being two levels deep

Reply:
I had a conversation with @Lou1415926 about this and we concluded that security group of the environment is not a part of `VPC` but it's a part of environment itself so it shouldn't go under `VPC` config and be on it's own in the env manifest, hence I made changes accordingly. 
What are your thoughts on this?

Reply:
As per internal discussion with the team we are concluding this as -

security_group should be inside `network.vpc` as `EnvironmentSecurityGroup` will get applied to all the services in the environment in the particular vpc. And it also aligns well with the `http.public.security_groups`  that we have in the env manifest. So `network.vpc`'s security_group is for all the services in the environment and `http.public`'s security_groups is applied to the load balancer.